src/www/ui/page/UploadPermissionPage.php

```diff
@@ -114,7 +125,8 @@ protected function handle(Request $request)
             'newPerm' => $newperm,
             'newGroup' => $newgroup,
             'uploadList' => $UploadList,
-            'permNames' => $GLOBALS['PERM_NAMES']
+            'permNames' => $GLOBALS['PERM_NAMES'],
+            'schedulerStatus' => $response_from_scheduler
```

Comment:
Please use the existing `message` block.

```suggestion
            'message' => $response_from_scheduler
```

Reply:
fixed the code